content/docs/reactivesearch/vue-searchbox/quickstart.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'QuickStart'
+meta_title: 'QuickStart to Vue SearchBox'
+meta_description: 'Vue SearchBox is a lightweight vue search library with some common utilities.'
+keywords:
+    - quickstart
+    - vue-searchbox
+    - search library
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'vue-searchbox-reactivesearch'
+---
+
+[vue-searchbox](https://github.com/appbaseio/vue-searchbox) - A lightweight Vue search library with some common utilities.
```

Comment:
@ShahAnuj2610 Update description. The `React` doc for getting started also needs an updated description.